pkg/service/healthserver/healthserver.go

```diff
@@ -189,6 +191,13 @@ func (h *httpHealthHTTPServerFactory) newHTTPHealthServer(port uint16, svc *Serv
 
 		if err := srv.ListenAndServe(); err != http.ErrServerClosed {
 			svc := srv.loadService()
+			if strings.Contains(err.Error(), unix.EADDRINUSE.Error()) {
```

Comment:
Please use `errors.Is(err, unix.EADDRINUSE)` here. It's a Go 1.13 addition to check for inner errors.

Reply:
Will address this.

Reply:
Done.